ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -118,3 +118,31 @@ def test_tensorflow_shuffle(
         value=values[0],
         seed=seed,
     )
+
+
+# random poisson
+@handle_frontend_test(
+    fn_tree='tensorflow.random.poisson',
+    dtype_value=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+    seed=helpers.ints(min_value=0, max_value=10),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_poisson(
+    frontend,
+    fn_tree,
+    on_device,
+    dtype_value,
+    seed,
+    test_flags,
+):
+    input_dtypes, values = dtype_value
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
+        value=values[0],
+        seed=seed,
+    )
```

Comment:
Please have a look at `test_poisson` to use as a reference https://github.com/unifyai/ivy/blob/e4cf0c7f7783e4a209b5291b5fb0bea2c0db621c/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_random.py#L197
You need to generate and pass all the function arguments in the test.

Reply:
If you're having problems with setting up your IDE, you could ask for help on discord.

Reply:
/usr/bin/python3 /opt/.pycharm_helpers/pycharm/_jb_pytest_runner.py -- /home/qalmaqihir/Downloads/Projects/ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py::test_tensorflow_poisson 
Testing started at 17:48 ...
Launching pytest with arguments ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py::test_tensorflow_poisson --no-header --no-summary -q in /ivy

============================= test session starts ==============================
collecting ... collected 0 items

============================ no tests ran in 0.00s =============================
ERROR: file or directory not found: ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py::test_tensorflow_poisson


Process finished with exit code 4

Empty suite

The error i get when I run the test function. I used the docker setting which was quite easy to follow from the tutorial video on docs